pkg/apis/clusterlink.net/v1alpha1/instance_types.go

```diff
@@ -96,7 +96,7 @@ type IngressSpec struct {
 	// +kubebuilder:default=none
 	// Type represents the type of service used to expose the ClusterLink deployment.
 	// Supported values: "LoadBalancer","NodePort", "none".
-	// The service name will be "cl-external-svc".
+	// The service name will be "cl-ingress".
```

Comment:
I think the addition of `-ingress` is a somewhat redundant (somewhat akin to as adding `-ns` to namespace names)
```suggestion
	// The service name will be "clusterlink".
```